cloudshell/cli/service/cli_service_helper.py

```diff
@@ -0,0 +1,66 @@
+class SendCommandWithRetries(object):
+    """
+    Help to execute command with retries.
+
+    If command execution raise an exception it helps
+    to reconnect or create a new session from the list of
+    defined sessions, optional.
+    """
+
+    MAX_RECREATE_RETRIES = 3
+    MAX_RECONNECT_RETRIES = 0
+    RECONNECT_TIMEOUT = 30
+
+    def __init__(
+        self,
+        cli_configurator,
+        command_mode,
+        logger,
+        recreate_retries=None,
```

Comment:
```suggestion
        recreate_retries=MAX_RECREATE_RETRIES,
```
you can insert default value here